thaw/src/field/field.css

```diff
@@ -12,6 +12,11 @@
     color: var(--colorNeutralForeground1);
 }
 
+.thaw-field__label--required {
+    padding: 0 var(--spacingHorizontalXS);
+    color: var(--colorPaletteRedForeground1);
```

Comment:
Modify to the following style to ensure consistency with FluentUI.

```css
padding-left: var(--spacingHorizontalXS);
color: var(--colorPaletteRedForeground3);
```

Reply:
done